kitsune/settings.py

```diff
@@ -84,6 +84,7 @@ def path(*parts):
 SERVER_EMAIL = config('SERVER_EMAIL', default='server-error@support.mozilla.org')
 
 PLATFORM_NAME = platform.node()
+K8S_DOMAIN = config('K8S_DOMAIN', default='')
```

Comment:
Will this include the "app name" (I guess k8s "namespace" is what it's called now)?

Reply:
with k8s `platform.node()` returns more meaniful answers like `sumo-dev-web-1092229904-8xjsl`. So I was planning `K8S_DOMAIN` to be the domain like, `frankfurt.moz.works`

Reply:
Also updated with k8s app settings to include `k8s_domain`

Reply:
Ah! cool. thanks for the info.